### DISCUSSION TITLE
Duda Ejercicio 2 - Actividad 8

### DISCUSSION THREAD

**Discussion by pipen77** [Taller-Abierto-de-Humanidades-Digitales/curso_python_extenso#72] [Dudas del curso]
Hola Jairo,

¿Me queda la duda de si en esta instrucción solo se debe tener como parámetro la cadena con el nombre o si también deberíamos tener en cuenta el diccionario como parámetro?

> La función fecha_nacimiento() recibe como parámetro un string con el nombre del autor. La función debe regresar la fecha de nacimiento del autor.

**Comment by jairomelo** [comment]
Hola @pipen77 ,

Efectivamente, recibe dos parámetros, nombre de autor y el diccionario. Estoy ajustado las instrucciones para que quede marcada esta corrección.

¡Mil gracias!

**Reply by pipen77** [reply]
Gracias Jairo. 👍

**Reply by pipen77** [reply]
En el ejercicio 3b. ¿La idea es que le la función reciba una nueva lista con los nombres de 1 o varios autores y que calcule la edad promedio o que tome todos los que están en el diccionario?. Si es al último caso ¿No sobra pedir la lista?

> La función edad_promedio() recibe como parámetros una lista con los nombres de los autores y un diccionario. La función debe regresar la edad promedio de los autores al momento de publicar su obra principal (int).

**Reply by jairomelo** [reply]
Puede ser, aunque si la lista se construye previamente (si se guarda en una variable con el resultado de ejecutar la función `autores()`) entonces es más económico invocar la lista en lugar de procesar nuevamente el diccionario. De todas maneras, funcionaría sin la lista, efectivamente. Esa ya es tu decisión de diseño 😉
